action.yml

```diff
@@ -117,7 +117,7 @@ runs:
         echo "post-build-hook = ${{ github.action_path }}/write-built.sh" | sudo tee -a /etc/nix/nix.conf
         mkdir -p "$HOME/.local/bin"
         cp '${{ github.action_path }}/save-built.sh' "$HOME/.local/bin/save-built"
-        echo "PATH='$PATH:$HOME/.local/bin'" > "$GITHUB_ENV"
```

Comment:
Did the quotes break something here? Assuming `GITHUB_ENV` runs with `eval` the quotes avoid space separation bugs.

Reply:
GitHub actions treates quotes literally for variables set in GITHUB_ENV,
so don't put quotes in PATH. Literal quotes in PATH appear to break
finding binaries on macOS

I'm guessing GitHub sets values literally and doesn't do an eval. You can see how it puts a literal quote in PATH, which I'm echoing, here: https://github.com/flox/install-flox-action/actions/runs/6238963480/job/16935871509#step:4:13

Reply:
Good to know.